guides/common/modules/proc_deploying-a-custom-ssl-certificate-to-hosts.adoc

```diff
@@ -1,14 +1,58 @@
 [id="deploying-a-custom-ssl-certificate-to-hosts_{context}"]
-
 = Deploying a Custom SSL Certificate to Hosts
 
-After you configure {ProductName} to use a custom SSL certificate, you must also install the `katello-ca-consumer` package on every host that is registered to this {ProductName}.
+After you configure {Project} to use a custom SSL certificate, you have to provide this certificate used by `subscription-manager` on hosts registered to {Project}.
```

Comment:
Suggestion: "After you configure {Project} to use a custom SSL certificate, you must deploy the certificate to hosts registered to {Project}."

I don't think you need to mention 'subscription-manager' here because it is not directly connected with the procedure. If it is important, I recommend mentioning it at the beginning of the "Configuring {Project} Server with a Custom SSL Certificate" assembly.

Reply:
Applied.

FYI: You can click the "suggestion" button (left button when writing a comment) to add your suggestion as diff.